features/home/claim-form/inputs/select-vesting.tsx

```diff
@@ -19,6 +19,7 @@ export const SelectVesting: FC<ClaimInputProps> = ({
   children,
 }) => {
   const ref = useRef<HTMLSpanElement>(null);
+  const orderedVestings = useMemo(() => options?.slice()?.reverse(), [options]);
```

Comment:
Maybe it makes sense to sort the vestings in the place where we get them from the contract?

Reply:
> Feels like it is purely presentation logic, here we want to sort them like this, there we will want to sort them somehow differently 🤔 I guess it would make sense to move this to helper or smth like this, but I don't think we need to alter the data

I doubt that unsorted data will be needed somewhere in the application. So why not do it right away?